src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,149 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+        tail = new Node<>(null, null, null);
+        head = new Node<>(null, null, null);
+    }
```

Comment:
```suggestion

```
by default tail and head default, and you can remove all this constructor, cause it's empty and we don't need to initialize here anything